src/core/aliases.zsh

```diff
@@ -12,6 +12,7 @@ _fzf_complete_enable_aliases() {
         if [[ -n $completer ]]; then
             eval "
                 _fzf_complete_$name() {
+                    source ${@[(r)*completers/$completer.zsh]}
```

Comment:
I don't like this function sources every time this complete function is called. How about just embedding the content of that function?

Reply:
Does it mean to insert `$(declare -f _fzf_complete_$completer)`?

Reply:
No. `$(cat -- $[@(r)*completers/$completer.zsh]})`